features/text-stroke-fill.yml

```
@@ -0,0 +1,10 @@
+name: Text stroke and fill
+description: The `-webkit-text-stroke-width` and `-webkit-text-stroke-color` CSS properties set the thickness and color of text outlines. The `-webkit-text-fill-color` sets the color within text character outlines. Both default to the text `color`.
```

```suggestion
description: The `-webkit-text-stroke-width` and `-webkit-text-stroke-color` CSS properties set the thickness and color of text outlines. The `-webkit-text-fill-color` sets the color within text character outlines. Both default to the text color.
```
Since it's technically `currentColor`, I think without the ticks makes sense here.